lib/cli-engine/cli-engine.js

```diff
@@ -1001,7 +1001,7 @@ class CLIEngine {
     /**
      * Returns the formatter representing the given format or null if the `format` is not a string.
      * @param {string} [format] The name of the format to load or the path to a
-     *      custom formatter.
```

Comment:
Why is it changing this? We've had this convention since the project started and I'd like to keep it. Is there a way to configure this so we don't lose our original formatting?